modules/olm-arch-catalog-operator.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-arch.adoc
+
+[id="olm-arch-catalog-operator_{context}"]
+= Catalog Operator
+
+The Catalog Operator is responsible for resolving and installing CSVs and the
+required resources they specify. It is also responsible for watching
+CatalogSources for updates to packages in channels and upgrading them
+(optionally automatically) to the latest available versions.
```

Comment:
It's not clear to me what `(optionally automatically)` means. If its still accurate, maybe:

> It is also responsible for watching CatalogSources for updates to packages in channels and upgrading them to the latest available versions. You have the option to upgrade automatically.